docs/guides/noise-learning.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1b2370f0-a4ee-49b1-9e53-e4527d37bc6f",
+   "metadata": {},
+   "source": [
+    "# Noise learning helper\n",
+    "\n",
+    "The error mitigation techniques [PEA](./error-mitigation-and-suppression-techniques#probabilistic-error-amplification-pea) and [PEC](./error-mitigation-and-suppression-techniques#probabilistic-error-cancellation-pec) both utilize a noise learning component based on a [Pauli-Lindblad noise model](https://arxiv.org/abs/2201.09866), which is typically managed during execution after submitting one or more jobs through `qiskit-ibm-runtime`. However, versions of `qiskit-ibm-runtime` 0.27.1 and later include a [`NoiseLearner`](../api/qiskit-ibm-runtime/noise_learner) and associated  [`NoiseLearnerOptions`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.NoiseLearnerOptions) class to obtain the results of these noise learning experiments which can then be stored as a `NoiseLearnerResult`. This page provides an overview of its usage and the associated options available.\n",
```

Comment:
> However, versions of `qiskit-ibm-runtime` 0.27.1 and later include a [`NoiseLearner`](../api/qiskit-ibm-runtime/noise_learner) and associated [`NoiseLearnerOptions`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.NoiseLearnerOptions) class to obtain the results of these noise learning experiments which can then be stored as a `NoiseLearnerResult`

This sentence is throwing me off a little bit. maybe it's the use of the word "however". I dont really understand what point you're makeing - are you telling me this is a new feature I can use? is this a good thing? why should i care?

Reply:
Just updated the wording of the intro sentences. Hopefully this is clearer now!